opcodes/riscv-opc.c

```diff
@@ -828,6 +828,339 @@ const struct riscv_opcode riscv_opcodes[] =
 {"clmulh",     0, INSN_CLASS_ZBC,  "d,s,t",    MATCH_CLMULH, MASK_CLMULH, match_opcode, 0 },
 {"clmulr",     0, INSN_CLASS_ZBC,  "d,s,t",    MATCH_CLMULR, MASK_CLMULR, match_opcode, 0 },
 
+/* RVP instructions */
+{"add8",        0, INSN_CLASS_ZPN, "d,s,t",     MATCH_ADD8, MASK_ADD8, match_opcode, 0 },
+{"add16",       0, INSN_CLASS_ZPN, "d,s,t",     MATCH_ADD16, MASK_ADD16, match_opcode, 0 },
+{"add64",       0, INSN_CLASS_ZPSF, "nds_rdp,nds_rsp,nds_rtp", MATCH_ADD64, MASK_ADD64, match_opcode, 0 },
+{"ave",         0, INSN_CLASS_ZPN, "d,s,t",     MATCH_AVE, MASK_AVE, match_opcode, 0 },
+{"bitrev",      0, INSN_CLASS_ZPN, "d,s,t",     MATCH_BITREV, MASK_BITREV, match_opcode, 0 },
+{"bitrevi",     0, INSN_CLASS_ZPN, "d,s,l",     MATCH_BITREVI, MASK_BITREVI, match_opcode, 0 },
+{"bpick",       0, INSN_CLASS_ZPN, "d,s,t,nds_rc", MATCH_BPICK, MASK_BPICK, match_opcode, 0 },
```

Comment:
nds_rc => r

Reply:
It looks to me that `nds_rc` can be replaced by `r` with the patch.
```
diff --git a/opcodes/riscv-dis.c b/opcodes/riscv-dis.c
index e6b5cba205..f20dc3db64 100644
--- a/opcodes/riscv-dis.c
+++ b/opcodes/riscv-dis.c
@@ -306,6 +306,11 @@ print_insn_args (const char *d, insn_t l, bfd_vma pc, disassemble_info *info)
          print (info->stream, "%s", riscv_gpr_names[rs1]);
          break;
 
+       case 'r':
+         print (info->stream, "%s",
+                riscv_gpr_names[EXTRACT_OPERAND (RS3, l)]);
+         break;
+
```

The regression test passes, but I am not sure about whether this change would affect the behaviour of .insn directive. Hi @Nelson1225 , any suggestion on it?